lib/models/command.js

```diff
@@ -237,9 +239,15 @@ let Command = CoreObject.extend({
 
     this._currentTask = task;
 
-    return Promise.resolve(task.run(options)).finally(() => {
-      delete this._currentTask;
-    });
+    return Promise.resolve(task.run(options))
+      .catch(error => {
+        logger.info(`An error occurred running \`${name}\` from the \`${this.name}\` command.`, error.stack);
```

Comment:
one of these guys? https://github.com/ember-cli/ember-cli/blob/de200a2dd57cba1542891fbb53672e66c062f864/lib/cli/cli.js#L265-L274